lib/ts/recipe/emailpassword/api/implementation.ts

```diff
@@ -555,9 +563,11 @@ export default function getAPIImplementation(): APIInterface {
                 return doUpdatePassword(new RecipeUserId(userIdForWhomTokenWasGenerated));
             }
         },
+
         signInPOST: async function ({
```

Comment:
if there is no session:
- we do what we do now

else if there is a session:
- we get the tenantids that belong to the session user
- if the input email exists in the current tenant, then what we do now for sign in is fine.
- if it exists in a different tenant, we call sign in from that tenant, and then we associate the other recipeUserId to the current tenant.

Reply:
@porcellus